loadtest/loadtest.go

```diff
@@ -73,12 +73,11 @@ func (lt *LoadTester) addUser() error {
 	}
 	lt.status.NumUsers++
 	lt.status.NumUsersAdded++
-	// Randomly login with the same user again, to simulate multiple sessions
-	// from the same user.
-	nextUser := rand.Int()%2 == 0
-	userID := activeUsers
-	if nextUser {
-		userID++
+	userID := activeUsers + 1
+	// If specified by the config, we login with the same user again,
+	// to simulate multiple sessions.
+	if lt.config.UsersConfiguration.UseMultipleSessionsPerUser && rand.Int()%2 == 0 {
```

Comment:
Does this mean we have a 50% chance of running a user twice? 
I'd like to control this with something like a `MultipleSessionsProbability` to `UsersConfiguration`?
Alternatively we can define a `AvgSessionsPerUser` (2 in this case) which would have a similar result but might be easier to understand/visualize.

Reply:
That's what I asked for here:
> Let me know if you want to randomness to be more skewed towards
creating new users rather than existing users. Right now its 50-50.

:smile:

Reply:
Lost that one, apologize :man_facepalming: